content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -295,6 +295,70 @@ a Pod is considered ready, see [Container Probes](/docs/concepts/workloads/pods/
 
 Please note that this field only works if you enable the `StatefulSetMinReadySeconds` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
 
+## PersistentVolumeClaim retention
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+The optional `.spec.persistentVolumeClaimRetentionPolicy` field controls if
+and how PVCs are deleted during the lifecycle of a StatefulSet. You must enable the
+`StatefulSetAutoDeletePVC` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+to use this field. Once enabled, there are two policies you can configure for each
+StatefulSet:
+
+`whenDeleted`
+: configures the volume retention behavior that applies when the StatefulSet is deleted
+
+`whenScaled`
+: configures the volume retention behavior that applies when the replica count of
+  the StatefulSet   is reduced; for example, when scaling down the set.
+  
+For each policy that you can configure, you can set the value to either `Delete` or `Retain`.
+
+`Delete`
+: The PVCs created from the StatefulSet `volumeClaimTemplate` are deleted, for
+  each Pod affected by the policy. With the `whenDeleted` policy, as all Pods are
+  deleted when the Statefulset is deleted, all PVCs from the `volumeClaimTemplate`
+  are deleted.
+  With the `whenScaled` policy, only PVCs corresponding to Pod replicas being scaled
```

Comment:
Just a note about the hugo markdown rederer, if you'd like this "With the `whenScaled`..." line to show up on the new line, you'll need to add another carriage return before it. Otherwise the renderer will put it in the same paragraph as the "The PVCs created..."

```suggestion

  With the `whenScaled` policy, only PVCs corresponding to Pod replicas being scaled
```

Reply:
I think if you did add the extra blank line, this would end the `<dd>` and `<dl>` elements in the final Markdown.
Another way to get a line break (if desired - and I don't think any is strictly needed) is to ensure that the line:
```markdown
  are deleted.
```
ends with **exactly two** spaces.

Reply:
Thanks, I don't think the line break is needed, I've reflowed it to make it more clear.